examples/09_svcmulti/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+include_directories(${AREG_EXAMPLES}/09_service/)
+include(${myDir}/CMakeLists.txt)
```

Comment:
Perhaps try starting using file glob :-)
Hopefully, something like this can work for us
```
file(GLOB files "${myDir}/*")
list(APPEND SRC ${files})
```

Reply:
At first, even for including core files, using `file (GLOB ..)` method was one of the easy method to just add everything together and indeed it did work. Although this method was discouraged by many, because even though it is an easy approach to add CMake to project, it lacks many benefit of an explicit source list.

Based on same rule, I tried for an explicit approach. 

I must say, there's no absolutely **any force** for using specific method. If you believe using `file(GLOB)` has more benefits for our situation, implementing such idea  wouldn't be  a problem. :)